src/Sarif.Viewer.VisualStudio/Telemetry/FeedbackTelemetryEvent.cs

```diff
@@ -42,6 +43,11 @@ public static void SendFeedbackTelemetryEvent(FeedbackModel feedback)
                 properties.Add(Snippet, JsonConvert.SerializeObject(feedback.Snippets));
             }
 
+            if (feedback.SarifLog != null)
+            {
+                properties.Add(SarifLog, JsonConvert.SerializeObject(feedback.SarifLog));
```

Comment:
>JsonConvert.SerializeObject(feedback.SarifLog) [](start = 41, length = 46)

should we trim for 8k characters?

Reply:
tried send a log more than 8k, the api call succeeded without error.
should we trim or let App Insight trim?

---
In reply to: [563573643](https://github.com/microsoft/sarif-visualstudio-extension/pull/353#discussion_r563573643) [](ancestors = 563573643)

Reply:
Can you check in AppInsights and see if it trimmed automatically? I would trim here, since we would need to send less data.

---
In reply to: [564029145](https://github.com/microsoft/sarif-visualstudio-extension/pull/353#discussion_r564029145) [](ancestors = 564029145,563573643)

Reply:
done